charts/eks-anywhere-packages/templates/job.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: init-job
+  namespace: eksa-packages
+spec:
+  template:
+    spec:
+      restartPolicy: OnFailure
+      serviceAccountName: {{ include "eks-anywhere-packages.serviceAccountName" . }}
+      containers:
+      - name: init-cronjob
```

Comment:
Change this name to init-job, this run doesn't have anything to do with the cronjob itself